CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 Full changelog coming soon. For now, here are the most important changes for doing major migrations:
 
+## Migrating to version 1.2
+- `OpenSeaPort::computeFees()` No longer accepts an `isPrivate` parameter.
```

Comment:
Think we should have something in the CHANGELOG for this, but not sure about new version #. Should we bump to 1.2 or 1.1.4?

Reply:
1.2? i would guess